modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -109,7 +109,7 @@ Node *EditorSceneImporterAssimp::import_scene(const String &p_path, uint32_t p_f
 	importer.SetPropertyBool(AI_CONFIG_PP_FD_REMOVE, true);
 	// Cannot remove pivot points because the static mesh will be in the wrong place
 	importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_PRESERVE_PIVOTS, false);
-	int32_t max_bone_weights = 4;
+	int32_t max_bone_weights = 8;
```

Comment:
:dango:

Reply:
3.2 should only still have 4 bone influences. Not sure about vulkan.

Reply:
Yeah I think we probably need to get support for 4.0 but we need to ask and see if we can get this

Reply:
reverted to 4 bone weights aiming for 3.2 if possible